build.gradle

```diff
@@ -36,30 +23,28 @@ repositories {
 }
 
 dependencies {
-    implementation("com.exactpro.th2:common:5.13.1-dev") {
-        exclude group: 'com.exactpro.th2', module: 'task-utils'
-    }
-    implementation "com.exactpro.th2:common-utils:2.2.3-dev"
-    implementation 'com.exactpro.th2:netty-bytebuf-utils:0.0.1'
-    implementation'com.exactpro.th2:conn-dirty-tcp-core:3.6.0-dev'
-    implementation 'com.exactpro.th2:grpc-lw-data-provider:2.3.1-dev'
+    implementation(libs.th2.common)
+    implementation(libs.th2.common.utils)
+    implementation(libs.th2.netty.bytebuf.utils)
+    implementation(libs.th2.conn.dirty.tcp.core)
+    implementation(libs.th2.grpc.lw.data.provider)
 
     implementation 'org.slf4j:slf4j-api'
-    implementation 'io.github.microutils:kotlin-logging:3.0.5'
+    implementation(libs.kotlin.logging)
     implementation 'org.apache.commons:commons-lang3'
 
     implementation 'io.netty:netty-all'
-    implementation 'com.google.auto.service:auto-service:1.1.1'
+    implementation(libs.auto.service)
```

Comment:
note: I think we don't need this dependency as runtime dependency - it can be use as `compileOnly` I think